tests/web_api/test_calculate.py

```diff
@@ -307,13 +307,48 @@ def test_periods(test_client):
 
     response_json = json.loads(response.data.decode('utf-8'))
 
-    yearly_variable = dpath.get(response_json, 'households/_/housing_tax')  # web api year is an int
+    yearly_variable = dpath.util.get(response_json, 'households/_/housing_tax')  # web api year is an int
     assert yearly_variable == {'2017': 200.0}
 
-    monthly_variable = dpath.get(response_json, 'households/_/housing_occupancy_status')  # web api month is a string
+    monthly_variable = dpath.util.get(response_json, 'households/_/housing_occupancy_status')  # web api month is a string
     assert monthly_variable == {'2017-01': 'tenant'}
 
 
+def test_two_periods(test_client):
+    '''
+    Made to test the else of 'if computation_results == {}:' in handlers.py->calculate
```

Comment:
Describe the use case instead of the code? If the code evolves, we might loose the object of this comment.

Reply:
Thanks, comment added.